packages/siri-sx-generator/__snapshots__/index.test.ts.snap

```diff
@@ -344,6 +394,31 @@ exports[`SIRI-SX Generator > correctly generates SIRI-SX XML without Version or
                                 <JourneyPlanner>false</JourneyPlanner>
                             </Blocking>
                         </Consequence>
+                        <Consequence>
+                            <Condition>unknown</Condition>
+                            <Severity>severe</Severity>
+                            <Affects>
+                                <Networks>
+                                    <AffectedNetwork>
+                                        <VehicleMode>rail</VehicleMode>
+                                        <AffectedLine>
+                                            <AffectedOperator>
+                                                <OperatorRef>NOC</OperatorRef>
+                                                <OperatorName>Test</OperatorName>
+                                            </AffectedOperator>
+                                            <LineRef>Line</LineRef>
+                                            <PublishedLineName>Line</PublishedLineName>
```

Comment:
PublishedLineName is being generated here when we don't expect it. might need checking whether the feature flag is being set correctly.

Reply:
fixed